components/brave_ads/browser/notification_helper_android.cc

```diff
@@ -28,6 +29,11 @@ bool NotificationHelperAndroid::IsNotificationsAvailable() const {
 // Starting in Android 8.0 (API level 26), all notifications must be
 // assigned to a channel.
 bool NotificationHelperAndroid::IsBraveAdsChannelEnabled() const {
+  char osVersion[PROP_VALUE_MAX+1];
+  int os_version = __system_property_get("ro.build.version.release", osVersion);
+  if (os_version <= 22) {
+    return true;
```

Comment:
What are the side effects of returning true on older devices? Have we considered elsewhere in the code that channels may not exist on all OS versions

Reply:
@tmancey I guess we just fall into sending notification. We don't need the channel on older devices.